test/ducks/apps/components/__snapshots__/permissionsList.spec.js.snap

```diff
@@ -1,94 +1,19 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`MyApplications component should be rendered correctly with konnector permissions 1`] = `
-<div>
-  <ReactMarkdownWrapper
-    source="In order to improve the value of your Cozy (cozytest.mock.cc), the application Mock requires access to the following data:"
-  />
-  <ul
-    className="sto-perm-list"
-  >
-    <li
-      className="io.cozy.bank.operations"
-    >
-      <ReactMarkdownWrapper
-        source="__io.cozy.bank.operations__"
-      />
-    </li>
-    <li
-      className="io.cozy.bills"
-    >
-      <ReactMarkdownWrapper
-        source="__Bills__"
-      />
-    </li>
-    <li
-      className="io.cozy.files"
-    >
-      <ReactMarkdownWrapper
-        source="__Files__"
-      />
-    </li>
-    <li
-      className="io.cozy.accounts"
-    >
-      <ReactMarkdownWrapper
-        source="__Accounts__"
-      />
-    </li>
-  </ul>
-</div>
+exports[`PermissionsList component should be rendered correctly with konnector permissions 1`] = `
+<ReactMarkdownWrapper
+  source="The application %{appName} doesn't require any data access to work correctly on your Cozy (cozytest.mock.cc)."
+/>
 `;
 
-exports[`MyApplications component should be rendered correctly with permissions 1`] = `
-<div>
-  <ReactMarkdownWrapper
-    source="In order to improve the value of your Cozy (cozytest.mock.cc), the application Mock requires access to the following data:"
-  />
-  <ul
-    className="sto-perm-list"
-  >
-    <li
-      className="io.cozy.files"
-    >
-      <ReactMarkdownWrapper
-        source="__Files__"
-      />
-    </li>
-    <li
-      className="io.cozy.apps"
-    >
-      <ReactMarkdownWrapper
-        source="__List of installed applications__"
-      />
-    </li>
-    <li
-      className="io.cozy.photos.albums"
-    >
-      <ReactMarkdownWrapper
-        source="__Photo albums__"
-      />
-    </li>
-    <li
-      className="io.cozy.contacts"
-    >
-      <ReactMarkdownWrapper
-        source="__Contacts__"
-      />
-    </li>
-    <li
-      className="io.cozy.settings"
-    >
-      <ReactMarkdownWrapper
-        source="__Settings__"
-      />
-    </li>
-  </ul>
-</div>
+exports[`PermissionsList component should be rendered correctly with permissions 1`] = `
+<ReactMarkdownWrapper
+  source="The application %{appName} doesn't require any data access to work correctly on your Cozy (cozytest.mock.cc)."
+/>
 `;
 
-exports[`MyApplications component should be rendered correctly without permissions 1`] = `
+exports[`PermissionsList component should be rendered correctly without permissions 1`] = `
 <ReactMarkdownWrapper
-  source="The application Mock doesn't require any data access to work correctly on your Cozy (cozytest.mock.cc)."
+  source="The application %{appName} doesn't require any data access to work correctly on your Cozy (cozytest.mock.cc)."
```

Comment:
All these snapshots is empty, can we do better snapshots here with mock permissions?